Readme.md

```diff
@@ -46,15 +46,111 @@ If you'd like to build your own VFS for Git Windows installer:
 * Create a folder to clone into, e.g. `C:\Repos\VFSForGit`
 * Clone this repo into the `src` subfolder, e.g. `C:\Repos\VFSForGit\src`
 * Run `\src\Scripts\BuildGVFSForWindows.bat`
-* You can also build in Visual Studio by opening `src\GVFS.sln` (do not upgrade any projects) and building. However, the very first 
+* You can also build in Visual Studio by opening `src\GVFS.sln` (do not upgrade any projects) and building. However, the very first
 build will fail, and the second and subsequent builds will succeed. This is because the build requires a prebuild code generation step.
 For details, see the build script in the previous step.
 
 The installer can now be found at `C:\Repos\VFSForGit\BuildOutput\GVFS.Installer\bin\x64\[Debug|Release]\SetupGVFS.<version>.exe`
 
+## Building VFS for Git on Mac
+
+Note that VFS for Git on Mac is under active development.
+
+* Ensure you have Xcode installed, have accepted the terms of use, and have launched Xcode at least once.
+
+* Install [Visual Studio for Mac ](https://visualstudio.microsoft.com/vs/mac). (This will also install the `dotnet` CLI).
+
+* If you still do not have the `dotnet` cli `>= v2.1.300` installed [manually install it]. You can check what version you have with `dotnet --version`.(https://www.microsoft.com/net/download/dotnet-core/2.1)
+
+* You will need to manage and sign your own certificate.
+
+  If you're using Xcode for the first time, you may have to login to Xcode with your Apple ID to generate a codesigning certificate. You can do this by launching Xcode.app, opening the PrjFS.xcworkspace and trying to build. You can find the signing options in the General tab of the project's settings.
+
+* Create a `VFSForGit` directory and Clone VFSForGit into a directory called `src` inside it:
+  ```
+  mkdir VFSForGit
+  cd VFSForGit
+  git clone https://github.com/Microsoft/VFSForGit.git src
+  cd src
+  ```
+
+* Prep your machine to use VFS for Git. The following are all done by the script below.
+  * install Homebrew
+  * install and setup the Git Credential Manager (with `brew`)
+  * install/update Java (with `brew`)
+  * install a VFS for Git aware version of Git
+
+  ```
+  Scripts/Mac/PrepFunctionalTests.sh
+  ```
+
+* From the src directory run
+
+  ```
+  Scripts/Mac/BuildGVFSForMac.sh [Debug|Release]
+  ```
+
+  _Troubleshooting if this fails_
+
+  If you get
+  ```
+  xcodebuild: error: SDK "macosx10.13" cannot be located.
+  ```
+  You may have the "XCode Command Line Tools" installed (helpfully by Mac OS) instead of full XCode.
+  Make sure
+  ```
+  xcode-select -p
+  ```
+
+  shows `/Applications/Xcode.app/Contents/Developer`. If it does not, install Xcode and then launch it (you can close it afterwards.)
+
+* For the time being, only for active development, you will have to disable the SIP (System Integrity Protection) in order to load the kext).

```

Comment:
I'd just remove the 'For the time being, only for active development' here.

Reply:
Not sure I agree. Most people will never need to rebuild the kext, just perhaps the user mode code. Lets err on the side of making people overly paranoid about disabling SIP.

Reply:
I agree that most people won't need/want to build the kext, but until we provide them with a properly codesigned PrjFSKext, they'll have to do that. IMO, whenever we get to the point where we publish a properly signed installer, we should update the docs.

Reply:
Yes but I'd rather have fewer people doing that rather than have anyone running with SIP disabled on accident. The statement here is accurate - right now, you cannot even use VFS for Git without disabling SIP, but that is just a moment in time limitation because we're in active development. This is not the desired state, and will stop being the case as soon as we publish a production-signed kext.